spec/models/visit_metrics_entry_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 RSpec.describe VisitMetricsEntry do
   it "requires mandatory fields" do
+    # TODO: Rewrite this spec or throw it away.
+
```

Comment:
I think, given this is in a metrics-related file, and we're going to be looking at (read: trashing) the metrics soon, this comment is probably unnecessary?